docs/pjrt/examples.md

```diff
@@ -1,12 +1,15 @@
 # PJRT Examples
 
+
+
```

Comment:
Thanks for writing this up!

Maybe reference the very basic example here?

https://github.com/openxla/xla/tree/main/xla/pjrt/plugin/example_plugin